addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -210,6 +211,9 @@ function factory(dependencies) {
          * hint `message-received`.
          */
         hasAutoScrollOnMessageReceived: attr(),
+        /**
+         * Last message in the context of the currently displayed thread cache.
+         */
```

Comment:
Similar aspect with `lastVisibleMessage` (dependent on current thread cache).
Would update comment of `lastVisibleMessage` too, because in context of PR was wondering why `unlink` on thread cache changed, which introduced the debate of rename, hence need clarification.